src/panels/developer-tools/action/developer-tools-action.ts

```diff
@@ -310,21 +295,20 @@ class HaPanelDevAction extends LitElement {
                           `component.${domain}.services.${serviceName}.fields.${field.key}.description`
                         ) || field.description}
                       </td>
-                      <td>
-                        ${this.hass.localize(
-                          `component.${domain}.services.${serviceName}.fields.${field.key}.example`
-                        ) || field.example}
-                      </td>
+                      <!-- prettier-ignore -->
```

Comment:
This seems unnecessary. Just let the formatter do its work

Reply:
Yes, as I stated in the PR description, when I take that out, the formatter changes things and I get unwanted whitespace above and below. Maybe that is something to do with my build settings though.

Reply:
Updated in 50bb0ff